Dockerfile

```diff
@@ -32,5 +30,7 @@ RUN TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1+PTX" TORCH_NVCC_FLAGS="-Xfatbin -compr
     CMAKE_PREFIX_PATH="$(dirname $(which conda))/../" \
     pip install -v .
 
+RUN git clone https://github.com/pytorch/vision.git && cd vision && pip install -v .
```

Comment:
Do we want this to be torchvision master, or should it be the latest stable release i.e. `conda install torchvision -c soumith` as this is a pytorch-dev dockerfile?